### PR TITLE
sudo: Drop groff build dependency.

### PR DESCRIPTION
**Pull request by mkosola**

### CODE REVIEW

File: rpm/sudo.spec [hunk 1]
```diff
@@ -57,7 +55,7 @@ rm -f %{buildroot}%{_sysconfdir}/sudoers.dist
 %files
 %defattr(-,root,root,-)
 %doc %{_docdir}/%{name}
-%doc %{_mandir}/man?/*
+%doc %{_mandir}/*
```
Comment (xfade) [inline rpm/sudo.spec]: Can't this be dropped completely?

**Review by xfade — COMMENTED** [review]
See comment in code

File: rpm/sudo.spec (hunk 1 above)
Comment (mkosola) [reply rpm/sudo.spec]: Dropped man pages completely

**Comment by xfade** [comment]
LGTM

**Pull request merged**